protocols/tests/test_migration/test_migration_reports_5_0_0_to_reports_4_0_0.py

```diff
@@ -0,0 +1,226 @@
+import factory.fuzzy
+from random import randint
+
+from protocols import reports_4_0_0
+from protocols import reports_5_0_0
+from protocols.util.dependency_manager import VERSION_61
+from protocols.util.dependency_manager import VERSION_500
+from protocols.util.dependency_manager import VERSION_400
+from protocols.util.factories.avro_factory import FactoryAvro
+from protocols.util.factories.avro_factory import GenericFactoryAvro
+from protocols.tests.test_migration.base_test_migration import TestCaseMigration
+from protocols.migration.migration_reports_5_0_0_to_reports_4_0_0 import MigrateReports500To400
+
+
+class ActionFactory(FactoryAvro):
```

Comment:
This can be removed. Only necessary to create mocked cancer report events in reports 4.0.0

Reply:
:+1: